masonry/README.md

```diff
@@ -34,7 +34,7 @@ Lots of things need improvements, e.g. text input is janky and snapshot testing
 The to-do-list example looks like this:
 
 ```rust
-use masonry::app_driver::{AppDriver, DriverCtx};
+use masonry::{AppDriver, DriverCtx};
```

Comment:
This wouldn't pass `cargo fmt` since it should be merged with a line below `use masonry::{Action, ...};`

Reply:
In practice cargo doesn't really enforce the merging style for imports.

Reply:
ah, I'm used to somewhere else where I enable that (non-stable) option. Good thing there aren't too many to fix up by hand!

Reply:
Yeah, I think I'll just run cargo fmt with that option and commit everything.

Reply:
You'll still have to get this one since it is in a doctest. :)